### DISCUSSION TITLE
`ctrl+space` does not show the intellisense in VSCode

### DISCUSSION THREAD

**Discussion by NoahELE** [community/community#56767] [Copilot Conversations]
### Select Topic Area

Bug

### Body

When I want the intellisense suggestion instead of the copilot auto completion, I usually use `ctrl+space` to show the popup panel, but after the update, `ctrl+space` does not show anything. I wonder if it is a bug.

**Comment by NoahELE** [comment]
I found that the shortcut `ctrl+i` can show the intellisense popup, but `ctrl+space` does not.

**Comment by nirav-gajera** [comment]
Check Keyboard Shortcuts: Open the Keyboard Shortcuts panel in Visual Studio Code by going to File > Preferences > Keyboard Shortcuts (or by pressing Ctrl+K followed by Ctrl+S). Search for "trigger suggest" in the search box and verify that the Ctrl+Space shortcut is assigned to the "editor.action.triggerSuggest" command.

**Reply by NoahELE** [reply]
I can confirm that I haven't change the keyboard shortcut of `ctrl+space`.

**Reply by nirav-gajera** [reply]
Check Language Mode: Ensure that you are in the correct language mode for the file you're editing.
or reset setting once

**Reply by NoahELE** [reply]
The language mode is correct I think, as `ctrl+i` shows the intellisense but `ctrl+space` does not.

**Reply by nirav-gajera** [reply]
Disable Conflicting Extensions: Some extensions might override or interfere with the default behavior of the Ctrl+Space shortcut. Disable any recently installed or suspect extensions one by one to see if the issue persists. You can do this by going to Extensions in the sidebar (Ctrl+Shift+X), finding the extension, and clicking the Disable button.
